src/app/containers/Timestamp/timestampUtilities.js

```diff
@@ -12,31 +13,21 @@ export const isValidDateTime = dateTime => {
 
 export const leadingZero = val => (val < 10 ? `0${val}` : `${val}`);
 
```

Comment:
It would be useful to add a comment here explaining that when using this function, we recommend using webpack configuration to only load in the relevant timezone, rather than all of `moment-timezone` https://github.com/bbc/simorgh/blob/f22671bea4b78d3bf186723535af8ae948c801ff/webpack.config.client.js#L126-L128